game-core/src/main/java/games/strategy/engine/GameEngineVersion.java

```diff
@@ -2,7 +2,7 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
-import games.strategy.util.Version;
+import org.triplea.util.Version;
```

Comment:
My guess is we have it to deal with when we had 5 version numbers but kept `Version.java` unchanged for compatibility. If so, that would make it legacy. For the purpose of evacuating `games.strategy.util`, it is one more thing to move.

Reply:
`GameEngineVersion` was introduced because `Version` is used for "versions" other than the game engine version.  The semantics of what constitutes a compatible engine or map version with another engine version are specific enough that they probably shouldn't be in the same general type that is also used for things like lobby version, etc.

However, that may no longer be the case since, with the recent changes `Version`'s implementation, it's becoming less generic.  In which case, anything else that happens to use it more like a general version type probably should be refactored to use a different type to avoid the coupling to what is effectively no longer a general utility class but rather something domain-specific.